tests/qnn/test_qnn_torch.py

```diff
@@ -25,6 +25,43 @@
 torch = pytest.importorskip("torch")
 
 
+pytestmark = pytest.mark.usefixtures("tape_mode")
+
+
+@pytest.fixture
+def get_circuit(n_qubits, output_dim, interface, tape_mode):
+    """Fixture for getting a sample quantum circuit with a controllable qubit number and output
+    dimension. Returns both the circuit and the shape of the weights."""
```

Comment:
This is temp copied here, as we need to add the `tape_mode` fixture, but still haven't ported the the TF tests